infrastructure/src/main/java/org/corfudb/infrastructure/AbstractServer.java

```diff
@@ -23,8 +29,19 @@
      *
      * @return A message handler.
      */
+    @Deprecated
     public abstract HandlerMethods getHandler();
 
+    /**
+     * Get the request handlers for this instance.
+     *
+     * @return The request handler methods.
+     */
+    public RequestHandlerMethods getHandlerMethods() {
+        //TODO: Make abstract once other servers are implemented.
+        return null;
+    }
```

Comment:
Its better to change this to abstract and add the stubs in the child classes. The child classes stubs can throw `UnsupportedOperationException` for their implementation.

Reply:
Done.